gradle.properties

```diff
@@ -17,5 +17,6 @@ org.gradle.jvmargs=-Xmx1536m
 android.useAndroidX=true
 # Automatically convert third-party libraries to use AndroidX
 android.enableJetifier=true
+android.nonTransitiveRClass=false
```

Comment:
enable this

Reply:
Why? Just for couple of ms build speed on average? 
Detail your review with reasons please

Reply:
1. They're enabled in all other repos, consistency
2. They're enabled by default anyway
3. Build speed gains
4. Future proofing the project